src/plugins/telemetry/server/routes/telemetry_usage_stats.ts

```diff
@@ -46,7 +46,8 @@ export function registerTelemetryUsageStatsRoutes(
       }
 
       const security = getSecurity();
-      if (security && unencrypted) {
+      // We need to check useRbacForRequest to figure out if ES has security enabled before making the privileges check
+      if (security && unencrypted && security.authz.mode.useRbacForRequest(req)) {
```

Comment:
I noticed that when running Kibana with `xpack.security.enabled: false`, users couldn't fetch the telemetry sample because it failed with 
```JSON
{"error":"no handler found for uri [/_security/user/_has_privileges] and method [POST]"}
```

Please, note that this didn't affect collecting telemetry because in that case it's encrypted and skips the security validation.